Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.h

```diff
@@ -182,6 +182,7 @@ namespace EMStudio
         void MotionEventTrackChanged(size_t eventNr, float startTime, float endTime, const char* oldTrackName, const char* newTrackName)            { UnselectAllElements(); CommandSystem::CommandHelperMotionEventTrackChanged(eventNr, startTime, endTime, oldTrackName, newTrackName); }
         void OnManualTimeChange(float timeValue);
         void toggleMotionEventPresetsPane();
+        void onRefreshSelection();
```

Comment:
update: I think as we're not overriding here this should be of the form `OnRefreshSelection` to be consistent with the coding standards